docs/resiliency/index.asciidoc

```diff
@@ -279,7 +279,7 @@ shard will be allocated upon reopening the index.
 [float]
 === Use two phase commit for Cluster State publishing (STATUS: DONE, v5.0.0)
 
-A master node in Elasticsearch continuously https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery-zen.html#fault-detection[monitors the cluster nodes]
+A master node in Elasticsearch continuously https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-fault-detection.html[monitors the cluster nodes]
```

Comment:
Due to the nature of the Resiliency page & the fact that we only publish one version of it, this is a case where linking to current makes sense.